lib/Conversion/TosaToTTIR/TosaToTTIRPatterns.cpp

```diff
@@ -200,7 +200,7 @@ class TosaToTTIRMaxPool2DOpConversionPattern
     rewriter.replaceOpWithNewOp<mlir::tt::ttir::MaxPool2dOp>(
         srcOp, TypeRange(outputTensor.getType()), adaptor.getInput(),
         outputTensor, dims[0], dims[1], strides[0], strides[1], 1, 1, false,
-        pad[2], pad[3], pad[0], pad[1]);
+        pad[2], pad[3], pad[0], pad[1], 1 /* channel_last */);
```

Comment:
I know you didn't write this `pad` part, but can you please add a comment/enum for this.

Reply:
Sure

Reply:
comments make it more messy... I've extracted each parameter into declaration which makes it more readable.